src/mono/wasm/testassets/WasmBasicTestApp/runtimeconfig.template.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
@radical Is there an explicity step needed to for this template to be transformed into runtimeconfig? I couldn't found anything, but when this app is built, the template is copied as is to the output

Reply:
Nevermind https://github.com/dotnet/runtime/blob/51af14c745e1074340a605faad3ba1bf8a969868/src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets#L64